msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-meeting.md

```diff
@@ -53,6 +53,100 @@ The following table provides the user types and lists the features that each use
 | Federated or External | Available | Not available | Not available | Available |
 | Anonymous | Not available | Not available | Not available | Not available |
 
+## Targeted in-meeting notification
```

Comment:
Adding as I noticed this while looking at the review site. On line 26 above (I'm unable to leave a comment there directly) it is not properly formatted json, and leads to the sample looking incorrect. Can we fix this?
![image](https://user-images.githubusercontent.com/6508687/214161393-969f2784-7e7a-412d-a34d-b93ee7c4593b.png)